vnext/templates/cpp-app/windows/ExperimentalFeatures.props

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Label="Microsoft.ReactNative Experimental Features">
+    <UseHermes>true</UseHermes>
+    <UseFabric>true</UseFabric>
+    <UseWinUI3>false</UseWinUI3>
+
+    <ReactExperimentalFeaturesSet>true</ReactExperimentalFeaturesSet>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Unpackaged {{ name }}" Condition="'$(SolutionName)'=='{{ name }}'">
+    <!--
+      WARNING - DO NOT MAKE THE VERSION OF MICROSOFT.UI.XAML BE ANYTHING BUT PRERELEASE.
+      Playground-Win32 is an unpackaged win32 app, and needs to use Microsoft.UI.Xaml from a prerelease,
+      to be able to carry WinUI in-app instead of using the Framework Package
+    -->
+    <WinUI2xVersion>2.7.0-prerelease.210913003</WinUI2xVersion>
+  </PropertyGroup>
```

Comment:
Given that the Fabric app isn't going to use WinUI, can this part go?

Reply:
REsolved.